presto-base-arrow-flight/src/test/java/com/facebook/plugin/arrow/TestingConnectionProperties.java

```diff
@@ -13,12 +13,21 @@
  */
 package com.facebook.plugin.arrow;
 
-public class TestingConnectionProperties
+public final class TestingConnectionProperties
```

Comment:
we can add `javax.annotation.concurrent.Immutable` annotation. No need to make the class final.

Reply:
done